packages/core/src/base/typography/link.scss

```diff
@@ -1,43 +1,61 @@
 /* stylelint-disable selector-class-pattern */
 @import '@cmsgov/design-system-support/src/settings/index';
 
-@if $ds-include-base-html-rulesets {
-  // <a> is the only base HTML element in the design system that
-  // has a style declaration. In all other cases, styles are applied using a
-  // namespaced class name. This selector isn't scoped under .ds-base, since that
```

Comment:
```
  // namespaced class name. This selector isn't scoped under .ds-base, since that	
  // would cause the selector's specificity to be higher than most other component	
  // selectors (i.e. ds-c-button), necessitating overly specific selectors anytime
```
Interestingly I'm noticing this exact issue with inverse `<a>` elements.
`a` is scoped under `.ds-base--inverse`, which is very specific, making it difficult to change the color of inverse link elements.

Reply:
This is an existing problem I believe. A good example of this is the links in the header:
<img width="795" alt="Screen Shot 2020-01-09 at 11 21 59 AM" src="https://user-images.githubusercontent.com/5424713/72102054-b0045a00-32eb-11ea-984d-16f9c786f466.png">
The styles from `c-header--title` are being overridden by `.ds-base--inverse a:visited`. I'm not entirely sure how to fix this, because we definitely want a way to default style links inside an inverse container.